python/example_code/greengrass/README.md

```diff
@@ -0,0 +1,80 @@
+# AWS IoT Greengrass code snippets
+
+## Purpose
+
+Shows how to use the AWS IoT Greengrass Core SDK to create AWS Lambda functions
+that publish MQTT messages, implement connectors, and retrieve secrets.
```

Comment:
Do we need to describe what MQTT means?

Reply:
Good question, but I think no. The Greengrass guide never expands it, and poking around online it looks like it's such a standard protocol that, for IoT, it's akin to HTTP and is pretty much never written out.